cilium-cli/connectivity/builder/from_cidr_host_netns.go

```diff
@@ -12,7 +12,7 @@ import (
 type fromCidrHostNetns struct{}
 
 func (t fromCidrHostNetns) build(ct *check.ConnectivityTest, templates map[string]string) {
-	newTest("from-cidr-host-netns", ct).
```

Comment:
Do we know why these three tests specifically are failing when run in parallel with other tests?